config/locales/client.en.yml

```diff
@@ -7,4 +7,5 @@ en:
   js:
     login:
       discourse_login:
-        name: "Discourse Login"
+        name: "Discourse ID"
+        title: "Login with Discourse ID"
```

Comment:
These are the cosmetic changes, consensus so far is to use Discourse ID as the label. Note that we don't need to rename the plugin because the plan is to move this authenticator to core soon.